data/i18n/pl/pl.toml

```diff
@@ -1,16 +1,16 @@
-# i18n strings for the Polish site.
-# NOTE: Please keep the entries in alphabetical order when editing
+# Stringi i18n dla polskiej wersji strony.
+# Uwaga: Zachowaj wpisy w kolejności alfabetycznej.
```

Comment:
(nit) Please leave the first English line as well.

Reply:
Sure, I'll revert those changes.